test/test-runtime-onMessage.js

```diff
@@ -239,4 +239,24 @@ describe("browser-polyfill", () => {
       });
     });
   });
+
+  it("ignores a certain chrome.runtime.lastError", () => {
+    const fakeChrome = {
+      runtime: {
+        lastError: {
+          message: "The message port closed before a response was received.",
```

Comment:
let's add a 'single line' inline comment which mention that this is the error defined as CHROME_SEND_MESSAGE_CALLBACK_NO_RESPONSE_MESSAGE in the polyfill internals.